### PR TITLE
Run auto start in CI

### PR DESCRIPTION
**Pull request by crazyhappygame**
THIS PROJECT IS IN DEVELOPMENT MODE. Any pull requests should merge to branch `dev`, otherwise will be rejected immediately.

## Checklist before requesting a review
-  [ ] I have performed a self-review of my code.
-  [ ] If it is a core feature, I have added thorough tests.
-  [ ] I have checked readme and add important infos to this PR (if it needed).
-  [ ] I have added/adapted some tests too.

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -39,6 +40,10 @@ jobs:
       - name: Build
         shell: pwsh
         run: .\build.ps1 -p win32 -a $env:BUILD_ARCH -dll $env:BUILD_DLL
+      - run: |
+          set AXMOL_START_AUTOTEST=YES
+          start /wait cpp_tests.exe
+        working-directory: build_x64/bin/Release
```
Comment (halx99) [inline .github/workflows/pr.yml:46]:
```suggestion
        working-directory: build_x64\bin\Release
```

File: .github/workflows/windows.yml [hunk 2]
```diff
@@ -31,6 +32,10 @@ jobs:
     - name: Build
       shell: pwsh
       run: .\build.ps1 -p win32 -a x64
+    - run: |
+        set AXMOL_START_AUTOTEST=YES
+        start /wait cpp_tests.exe
+      working-directory: build_x64/bin/Release
```
Comment (halx99) [inline .github/workflows/windows.yml:38]:
```suggestion
      working-directory: build_x64\bin\Release
```

**Review by halx99 — DISMISSED** [review]

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -39,6 +40,10 @@ jobs:
       - name: Build
         shell: pwsh
         run: .\build.ps1 -p win32 -a $env:BUILD_ARCH -dll $env:BUILD_DLL
+      - run: |
+          set AXMOL_START_AUTOTEST=YES
+          start /wait cpp_tests.exe
+        working-directory: build_x64\bin\Release
```
Comment (halx99) [inline .github/workflows/pr.yml:46]:
```suggestion
        working-directory: build_x64\bin\cpp_tests\Release
```

File: .github/workflows/pr.yml [hunk 4]
```diff
@@ -39,6 +40,10 @@ jobs:
       - name: Build
         shell: pwsh
         run: .\build.ps1 -p win32 -a $env:BUILD_ARCH -dll $env:BUILD_DLL
+      - run: |
+          set AXMOL_START_AUTOTEST=YES
+          start /wait cpp_tests.exe
```
Comment (halx99) [inline .github/workflows/pr.yml:45]:
```suggestion
          Start-Process  cpp_tests.exe -Wait
```

File: .github/workflows/windows.yml [hunk 5]
```diff
@@ -31,6 +32,10 @@ jobs:
     - name: Build
       shell: pwsh
       run: .\build.ps1 -p win32 -a x64
+    - run: |
+        set AXMOL_START_AUTOTEST=YES
+        start /wait cpp_tests.exe
```
Comment (halx99) [inline .github/workflows/windows.yml:37]:
```suggestion
        Start-Process  cpp_tests.exe -Wait
```

**Comment by rh101** [comment]
> seems the auto test cost more than 30min and cancelled by github action

Please do not merge this change in, as the impact on the CI build for something like this is not worth it in any way.

**Comment by halx99** [comment]
> > seems the auto test cost more than 30min and cancelled by github action
> 
> Please do not merge this change in, as the impact on the CI build for something like this is not worth it in any way.

ok

**Comment by halx99** [comment]
Close since don't want run it in github ci

**Pull request closed without merging**